### ISSUE TITLE
Bug in the function 'find_parcellation_cut_coords'

### ISSUE THREAD

**Issue by NeuroVR-Song** [nilearn/nilearn#3076]
When I extract coordinates for ROIs(labels), the 'find_parcellation_cut_coords' function returns the coordinates with the randomized order. That is, the order of the extracted coordinates is different from it of the input labels.
I think that's because of the 'set' function in below: https://github.com/nilearn/nilearn/blob/1607b52458c28953a87bbe6f42448b7b4e30a72f/nilearn/plotting/find_cuts.py#L429
Is this done on purpose?
Otherwise, I think it would be better to sort the order before returning the results.

### Expected behavior
![image](https://user-images.githubusercontent.com/71465920/144563770-83c459f6-cac0-4873-a0da-85e0ee26addd.png)

### Actual behavior
![image](https://user-images.githubusercontent.com/71465920/144563147-ad7814f3-3e8f-4f5f-8f2e-87b77ecec618.png)

### Steps and code to reproduce bug

```python

```

**Comment by NicolasGensollen** [comment]
Thanks for opening this @NeuroVR-Song 
I agree that sorting before returning the labels and coordinates would be better.
I think that we could simply replace the line you referenced with the set difference with:

```python
unique_labels = np.unique(labels_data)
unique_labels = np.delete(unique_labels, background_label)
```

WDYT?
Do you want to make a PR?

**Comment by NeuroVR-Song** [comment]
Yes, I am gonna make it :)

Best regards,
Myeong

**Comment by NicolasGensollen** [comment]
Closing as this was fixed in #3078
